test/models/chat_test.dart

```diff
@@ -33,7 +33,11 @@ void main() {
       senderId: profileId + 1,
       createDependencies: false,
     );
-    SupabaseManager.setCurrentProfile(profile);
+
+    setUp(() {
+      SupabaseManager.setCurrentProfile(profile);
```

Comment:
@istzen another error where `setUp` was missing, although I'm not sure why this did not happen before?

Reply:
thanks and sorry